roxctl/common/flags/timeout.go

```diff
@@ -18,16 +18,21 @@ func AddTimeoutWithDefault(c *cobra.Command, defaultDuration time.Duration) {
 
 // AddTimeout adds a timeout flag to the given command, with the global default value.
 func AddTimeout(c *cobra.Command) {
-	AddTimeoutWithDefault(c, 30*time.Second)
+	AddTimeoutWithDefault(c, 1*time.Minute)
 }
 
 // Timeout returns the set timeout.
 func Timeout(c *cobra.Command) time.Duration {
 	duration, err := c.Flags().GetDuration(timeoutFlagName)
-	if err != nil {
-		// This is a programming error. You shouldn't use the timeout flag unless you've added it to your command!
-		// This helps us fail explicitly instead of defaulting to a zero timeout and allowing people to think it worked.
-		panic(fmt.Sprintf("command does not have a timeout flag: %v", err))
+	if err == nil {
+		return duration
 	}
-	return duration
+
+	duration, err = c.PersistentFlags().GetDuration(timeoutFlagName)
+	if err == nil {
+		return duration
+	}
+	// This is a programming error. You shouldn't use the timeout flag unless you've added it to your command!
+	// This helps us fail explicitly instead of defaulting to a zero timeout and allowing people to think it worked.
+	panic(fmt.Sprintf("command does not have a timeout flag: %v", err))
```

Comment:
nit: Why not putting above comment into panic message. Once it happens you will get all required informations.

Reply:
On the other hand maybe we should use `Should` here as this is a runtime check that will fail if nobody execute the command.